src/main/ml-modules/ext/serviceModel.sjs

```diff
@@ -1061,6 +1061,7 @@ function getGeoServerLayerSchema(layerName) {
   return layer;
 }
 
+exports.MAX_RECORD_COUNT = MAX_RECORD_COUNT;
```

Comment:
This looks unnecessary - though it might be a nice future refactor as this is declared in both this module and geoQueryService. But it's not being used yet.